Content.Client/PDA/Ringer/RingtoneMenu.xaml.cs

```diff
@@ -0,0 +1,129 @@
+
+                if(!isNote(RingerNoteOneInput.Text))
+                {
+                    RingerNoteOneInput.Text = _previousNoteInputs[0];
+                }
+                else
+                {
+                    _previousNoteInputs[0] = RingerNoteOneInput.Text;
+                }
+
+                RingerNoteOneInput.CursorPosition = RingerNoteOneInput.Text.Length; //Resets caret position to the end of the typed input
+            };
+
+            //RingerNoteTwoInput
+            RingerNoteTwoInput.OnTextChanged += _ => //Prevents unauthorized characters from being entered into the LineEdit
+            {
+                RingerNoteTwoInput.Text = RingerNoteTwoInput.Text.ToUpper();
+
+                if(!isNote(RingerNoteTwoInput.Text))
+                {
+                    RingerNoteTwoInput.Text = _previousNoteInputs[1];
+                }
+                else
+                {
+                    _previousNoteInputs[1] = RingerNoteTwoInput.Text;
+                }
+
+                RingerNoteTwoInput.CursorPosition = RingerNoteTwoInput.Text.Length; //Resets caret position to the end of the typed input
+            };
+
+            //RingerNoteThreeInput
+            RingerNoteThreeInput.OnTextChanged += _ => //Prevents unauthorized characters from being entered into the LineEdit
+            {
+                RingerNoteThreeInput.Text = RingerNoteThreeInput.Text.ToUpper();
+
+                if(!isNote(RingerNoteThreeInput.Text))
+                {
+                    RingerNoteThreeInput.Text = _previousNoteInputs[2];
+                }
+                else
+                {
+                    _previousNoteInputs[2] = RingerNoteThreeInput.Text;
+                }
+
+                RingerNoteThreeInput.CursorPosition = RingerNoteThreeInput.Text.Length; //Resets caret position to the end of the typed input
+            };
+
+            //RingerNoteFourInput
+            RingerNoteFourInput.OnTextChanged += _ => //Prevents unauthorized characters from being entered into the LineEdit
+            {
+                RingerNoteFourInput.Text = RingerNoteFourInput.Text.ToUpper();
+
+                if(!isNote(RingerNoteFourInput.Text))
+                {
+                    RingerNoteFourInput.Text = _previousNoteInputs[3];
+                }
+                else
+                {
+                    _previousNoteInputs[3] = RingerNoteFourInput.Text;
+                }
+
+                RingerNoteFourInput.CursorPosition = RingerNoteFourInput.Text.Length; //Resets caret position to the end of the typed input
+            };
+        }
+
+        protected override DragMode GetDragModeFor(Vector2 relativeMousePos)
+        {
+            //Prevents the ringtone window from being resized
+            return DragMode.Move;
+        }
+
+        private bool isNote(string input) //Determines whether or not the characters inputed are authorized
```

Comment:
```suggestion
        private bool IsNote(string input)
```

Don't inline comments on a method.